usmqe/api/tendrlapi/tendrlapi.py

```diff
@@ -0,0 +1,214 @@
+"""
+Tendrl REST API.
+"""
+
+import requests
+import pytest
+from usmqe.api.base import ApiBase
+
+LOGGER = pytest.get_logger("tendrlapi", module=True)
+
+
+class ApiCommon(ApiBase):
+    """ Common methods for Tendrl REST API.
+    """
+
+    def login(self, username, password, asserts_in=None):
+        """ Login to REST API
+
+        Name:        "login",
+        Method:      "POST",
+        Pattern:     "auth/login",
+
+        Args:
+            username: username
+            password: password
+            asserts_in: assert values for this call and this method
+        """
+    pass
+
+    def logout(self, asserts_in=None):
+        """ Logout from REST API
+
+        Name:        "logout",
+        Method:      "POST",
+        Pattern:     "auth/logout",
+
+        Args:
+            asserts_in: assert values for this call and this method
+        """
+    pass
+
+
+class ApiGluster(ApiCommon):
+    """ Gluster methods for Tendrl REST API.
+    """
+
+    def get_nodes(self):
+        """ Get list node ids.
+
+        Name:        "get_nodes",
+        Method:      "GET",
+        Pattern:     "GetNodeList",
+        """
+        pattern = "GetNodeList"
+        response = requests.get(pytest.config.getini("usm_api_url") + pattern)
+        self.print_req_info(response)
+        self.check_response(response)
+        return response.json()
+
+    def import_cluster(self, cluster_data):
+        """ Import gluster cluster defined by json.
+
+        Name:        "import_cluster",
+        Method:      "POST",
+        Pattern:     "GlusterImportCluster",
+
+        Args:
+            cluster_data: json structure containing data that will be sent to api server
+        """
+        pattern = "GlusterImportCluster"
+        response = requests.post(pytest.config.getini("usm_api_url") + pattern,
+                                 json=cluster_data)
+        asserts = {
+            "reason": 'Accepted',
+            "status": 202,
+        }
+        self.print_req_info(response)
+        self.check_response(response, asserts)
+        return response.json()
+
+    def get_cluster_list(self):
+        pattern = "GetClusterList"
+        response = requests.get(pytest.config.getini("usm_api_url") + pattern)
+        self.print_req_info(response)
+        self.check_response(response)
+        return response.json()
+
+    def find_id_in_list(self, cluster_id, positive=True):
```

Comment:
I think that will be more readable if this wrapper will be unwrapped directly in test.